src/app/src/main/kotlin/org/vaccineimpact/api/app/controllers/TouchstoneController.kt

```diff
@@ -72,21 +75,71 @@ class TouchstoneController(context: ControllerContext) : AbstractController(cont
         val source = context.params(":source-code")
         val type = context.params(":type-code")
         val gender = context.queryParams("gender")
-        return repo.getDemographicData(type, source, touchstone.id, gender?: "both")
+        val format = context.queryParams("format")
+
+        val splitData = repo.getDemographicData(type, source, touchstone.id, gender ?: "both")
+
+        val tableData = when(format){
+
+            "wide" -> getWideDemographicDatatable(splitData.tableData.data)
+            "long" -> splitData.tableData
+            else -> throw BadRequest("Format '$format' not a valid csv format. Available formats are 'long' " +
+                    "and 'wide'.")
```

Comment:
Sorry to keep being picky about this but: We should still probably support a default value if the user doesn't choose a format at all, right? Does `queryParams` return `null` if the parameter hasn't been specified. If so the elvis operator (`?:`) would be the quick fix.

Reply:
Oh yeah, there's also a Teamcity test failing bc of this, you're totally right